CONTRIBUTING.md

```diff
@@ -0,0 +1,270 @@
+Containers organisation](https://github.com/kata-containers). Unless
+explicitly stated otherwise, all the Kata Containers repositories follow the
+process documented here.
+
+## Code of Conduct
+
+All contributors must agree to the project [code of conduct](CODE_OF_CONDUCT.md).
+
+## Golang Coding Style
+
+The usual Go style, enforced by `gofmt`, should be used. Additionally, the [Go
+Code Review](https://github.com/golang/go/wiki/CodeReviewComments) document
+contains a few common errors to be mindful of.
+
+## Certificate of Origin
+
+In order to get a clear contribution chain of trust we use the [signed-off-by
+language](https://ltsi.linuxfoundation.org/software/signed-off-process/)
+used by the Linux kernel project.
+
+## Pull requests
+
+All the repositories accept contributions by a github Pull request (PR).
+
+### Before starting work on a PR
+
+We welcome all contributions but to minimise the chance of multiple people
+working on the same feature or bug fixe (yes, it has happened), we recommend
+strongly that you raise a github issue **before** you start working on a PR.
+
+If you are a new contributor you will not be able to assign the issue to
+yourself so please raise an issue and add a comment stating that you intend to
+work on the issue yourself. This gives the team visibility of the work you
+plan to do.
+
+The other advantage to raising the issue at the outset is that our process requires an
+issue to be associated with every PR (see [patch format](#patch-format)).
+
+### Normal PR workflow
+
+Github has a basic introduction to the PR process
+[here](https://help.github.com/articles/using-pull-requests/).
+
+When submitting your PR, treat the PR message the same
+you would a patch message, including pre-fixing the title with a subsystem
+name. Github by default seems to copy the message from your first patch, which
+many times is appropriate, but please ensure your message is accurate and
+complete for the whole PR, as it ends up in the git log as the merge
+message.
+
+Your PR may get some feedback and comments, and require some rework.
+The recommended procedure for reworking is to rework your branch to a new clean
+state and "force push" it to your github branch. Github understands this action, and
+does sensible things in the online comment history. Do not pile patches on
+patches to rework your branch. Any relevant information from the github
+comments section should be re-worked into your patch set, as the ultimate place
+where your patches are documented is in the git log, and not in the github
+comments section.
+
+For more information on github "force push" workflows see
+[here](http://blog.adamspiers.org/2015/03/24/why-and-how-to-correctly-amend-github-pull-requests/).
+
+It is perfectly fine for your PR to contain more than one patch - use
+as many patches as you need to implement the Request (see the previously
+mentioned "small patch" thoughts). Each PR should only cover one
```

Comment:
is there a previous 'small patch' thought bit, in this doc?

Reply:
Another good catch - this is present in the CC runtime's version of this doc too. Removing...